csrc/index_compute.cpp

```diff
@@ -2238,6 +2238,7 @@ kir::TensorIndex* Index::getConsumerIndex(
     DataType as_type) {
   Val* index = nullptr;
   if (!ir_utils::hasRootToLoopLinearTransformations(consumer) ||
+      ir_utils::isCpAsyncBulkLoad(consumer->definition()) ||
```

Comment:
Encountered an indexing error. Don't why. Don't care why.

Reply:
For tracking purposes, what was the error?

Reply:
I believe it is this one: https://github.com/NVIDIA/Fuser/blob/bad9e50bc9539054050310f423317b4c1d259c53/csrc/index_compute.cpp#L1966-L1975

Reply:
That was my guess. I had to add a similar change to my matmul scheduler branch.